src/view/channel-detail.js

```diff
@@ -54,9 +54,14 @@ const ChannelDetailView = ({ store, nav }) => (
         <DetailField name="Balance">
           {store.selectedChannel.localBalanceLabel} {store.unitLabel}
         </DetailField>
-        <Button style={styles.deleteBtn} onPress={() => nav.goChannelDelete()}>
-          <ButtonText style={styles.deleteBtnText}>CLOSE CHANNEL</ButtonText>
-        </Button>
+        {/Close|Closing/i.test(store.selectedChannel.statusLabel) ? null : (
```

Comment:
Change to `/close|closing/i.test(store.selectedChannel.status)` as the labels are language dependent and will change once we start to translate the app into different languages.